packages/clay-layout/package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "@clayui/layout",
+	"version": "3.0.0-alpha.1",
+	"description": "ClayLayout component",
```

Comment:
nit, for the future: go over the descriptions in our packages. Doesn't feel like this says much about the package, what it does or why you'd want to download. Maybe we can get some copy from the Lexicon site and/or team

Reply:
One of the consequences of dividing a library into 45 packages is that you have to come up with 45 descriptions. 😀